keadm/cmd/keadm/app/cmd/edge/join_others.go

```diff
@@ -316,6 +317,10 @@ func createV1alpha1EdgeConfigFiles(opt *common.JoinOptions) error {
 		edgeCoreConfig.Modules.Edged.Labels = setEdgedNodeLabels(opt)
 	}
 
+	if opt.ImageRepository != "" {
+		edgeCoreConfig.Modules.Edged.PodSandboxImage = path.Join(opt.ImageRepository, constants.DefaultPodSandboxImage)
```

Comment:
Looks good, and now this seems to be set in runtime(containerd), the value in edged/kubelet is only used in imageGC, ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/config/flags.go#L53

cc @Shelley-BaoYue

Reply:
> Looks good, and now this seems to be set in runtime(containerd), the value in edged/kubelet is only used in imageGC, ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/config/flags.go#L53

Yes, I see the sendbox image is configured in containerd  or cri-dockerd. Do we let the user configure it themselves, or do we change the container configuration when executing the join command? @fisherxu @Shelley-BaoYue

Reply:
I think user configure it by themselves is better.  If we change the cri config and restart runtime, will the keadm join command become risk?  Or we can let the users to choose whether it can be configured by keadm.

Reply:
> I think user configure it by themselves is better. If we change the cri config and restart runtime, will the keadm join command become risk? Or we can let the users to choose whether it can be configured by keadm.

I agree that it is better to let users configure themselves now, and I will close this pr.